include/pel_handler.hpp

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include <iostream>
+#include <string>
+#include <unordered_map>
+
+namespace vpd
+{
+/**
+ * @brief Class to support logging PEL.
+ *
+ * This class provides framework to call 'create' method of 'phosphor-logging'
+ * service to log a PEL.
+ */
+class PelHandler
```

Comment:
Better name would be ErrorHandler. vpd::ErrorHandler

Basically 'p' stands for phosphor.  This is not phosphor error logging handler..

Same goes with the file name as well. error_handler.hpp/cpp

Reply:
Updated